mui/scripts/linkComponents.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+BLUE='\033[0;34m'
+BBLUE='\033[1;34m' #BOLD
+PURPLE='\033[0;35m'
+RED='\033[0;31m'
+BRED='\033[1;31m' #BOLD
+GREEN='\033[0;32m'
+BGREEN='\033[1;32m' #BOLD
+GRAY='\033[1;30m'
+NC='\033[0m' # No Color
+
+echo -e "${BLUE}Building components...${NC}"
+yarn build
+
+echo -en "${BLUE}Creating new folder in node_modules...${NC}"
+rm -rf "./demos/showcase/node_modules/@pxblue/icons-mui"
+mkdir -p "./demos/showcase/node_modules/@pxblue/icons-mui"
+# rm -rf "./demos/storybook/node_modules/@pxblue/icons-mui"
```

Comment:
Can we remove all of the 'storybook' comments in here...there is no storybook demo

Reply:
These comments are removed now